packages/medulas-react-components/src/components/Link/index.tsx

```diff
@@ -5,10 +5,22 @@ interface Props extends LinkProps {
   readonly children: React.ReactNode;
 }
 
-const Link = ({ children, to, ...rest }: Props): JSX.Element => (
-  <RouterLink to={to} {...rest}>
-    {children}
-  </RouterLink>
-);
+const Link = ({ children, to, ...rest }: Props): JSX.Element => {
+  if (typeof to === 'string') {
+    if (/^(mailto:)|((https?:)?\/\/)/.test(to)) {
```

Comment:
This also matches the string `xxxx//yyyyy`. I think the last `?` should not be there

Reply:
Removing the last `?` we won't accept `//foo.bar/`, I think based on our usage we can remove it. Thoughts?

Reply:
Ah, I see. The issue I found comes because the `^` only binds to the mailto alternative (`^` binds stronger than `|`). So adding addional brackets resolves it: `/^((mailto:)|((https?:)?\/\/))/`. This way, `//foo.bar/` is accepted as well